cmd/dcrdata/views/treasury.tmpl

```diff
@@ -61,22 +60,20 @@
             {{- end}}
             </span>
             <br>
-            {{/*  <span class="text-secondary fs16 lh1rem">{{intComma $bal.SpendCount}} inputs</span> */}}
+            <span class="text-secondary fs14 lh1rem">{{intComma $bal.SpendCount}} spends</span>
           </div>
           <span></span>
         </div>
         <div class="position-relative d-flex justify-content-between align-items-start flex-wrap">
           <span>See also the <a href="/address/{{$.DevAddress}}">legacy treasury</a> during the transition to this decentralized treasury account.</span>
         </div>
-        {{/*
         <div class="row pb-2">
-          {{- if ne .NumUnconfirmed 0}}
+          {{- if ne $bal.ImmatureCount 0}}
           <div class="col-12 pb-2 fs14 text-secondary text-left" data-target="address.numUnconfirmed">
-            <span class="font-weight-bold">Unconfirmed</span>: <span class="addr-unconfirmed-count">{{.NumUnconfirmed}}</span>
+            <span class="font-weight-bold">Unconfirmed</span>: <span class="addr-unconfirmed-count">{{$bal.ImmatureCount}}</span>
```

Comment:
I changed this to "Immature" in the text of the span

Reply:
Noticed that the `$TxnCount` seems to be showing zero:

![image](https://user-images.githubusercontent.com/9373513/117522952-c53ed680-af7b-11eb-94b4-e5b25220b608.png)

Reply:
That message is all messed up. It says this on page 2 now.

![image](https://user-images.githubusercontent.com/6109680/117524739-b876ae80-af8c-11eb-91b0-f8c5d15b9ed9.png)